### ISSUE TITLE
Non-standard port not working(Could possibly be stupidity on my part)

### ISSUE THREAD

**Issue by MethodOrMadness** [Pugmatt/BedrockConnect#184]
i tried setting the server port(v4, not v6 since my ISP doesnt use v6) to a non-default port, and when trying to connect it refuses to work

**Comment by kmpoppe** [comment]
Hi @MethodOrMadness, as far as I understand, 19132 is the default port _most_ consoles will try to connect to from the static server list. Thus, only if BedrockConnect is set to the default port, will (i.e.) the Nintendo Switch see the BedrockConnect server instead of the "featured" server.

Kai

**Comment by MethodOrMadness** [comment]
Thanks for the reply, i should be more clear, the steps taken are:

1. connect to bedrock connect by setting the DNS and joining a server
2. Typing in the serverIP and port
at this point when the client tries to connect to my server, it will say "unable to connect"

**Comment by kmpoppe** [comment]
Thanks for the clarification :)
So, if you've set your server's port to the default one you can connect? Is the Bedrock server on a hosted machine (cloud server)? Do you maybe need to punch a hole into that server hoster's firewall for the different port?

It's hard to diagnose these problems from afar, there are any number of reasons why this couldn't work beyond anything that's to do with BedrockConnect.

I for one have multiple servers sitting at different non-standard and that works fine, so my guess is that's something between your console and the server software.

Kai

**Comment by MethodOrMadness** [comment]
Yes, if the server's port is set to the default(19132) i can connect, anything else and it refuses per the previously mentioned error,

No the server is running locally on my network, all the ports are configured properly on UFW and my router(for clients outside the LAN)

**Comment by kmpoppe** [comment]
@MethodOrMadness does this issue still persist?
